[analyzegc is unhappy](https://buildkite.com/julialang/julia-master/builds/47672#0196e818-6b39-4e7a-b80a-9f6ccc868903/952-1294):
```
/cache/build/builder-amdci4-7/julialang/julia-master/src/init.c:560:39: error: Implicit Atomic seq_cst synchronization [concurrency-implicit-atomics,-warnings-as-errors]
  560 |     if (!jl_options.handle_signals && jl_n_threads > 1) {
      |                                       ^
make: *** [Makefile:571: clang-tidy-init] Error 1
```

--- a/src/init.c
+++ b/src/init.c
@@ -557,7 +557,7 @@ static void restore_fp_env(void)
     if (jl_set_zero_subnormals(0) || jl_set_default_nans(0)) {
         jl_error("Failed to configure floating point environment");
     }
-    if (!jl_options.handle_signals && jl_atomic_load_relaxed(&jl_n_threads) > 1) {
+    if (jl_options.handle_signals == JL_OPTIONS_HANDLE_SIGNALS_OFF && jl_atomic_load_relaxed(&jl_n_threads) > 1) {
         jl_error("Cannot use `--handle-signals=no` with multiple threads (JULIA_NUM_THREADS > 1).\n"
         "This will cause segmentation faults due to GC safepoint failures.\n"
         "Remove `--handle-signals=no` or set JULIA_NUM_THREADS=1.\n"

--- a/src/init.c
+++ b/src/init.c
@@ -557,7 +557,7 @@ static void restore_fp_env(void)
     if (jl_set_zero_subnormals(0) || jl_set_default_nans(0)) {
         jl_error("Failed to configure floating point environment");
     }
-    if (!jl_options.handle_signals && jl_n_threads > 1) {
+    if (!jl_options.handle_signals && jl_atomic_load_relaxed(&jl_n_threads) > 1) {
         jl_error("Cannot use `--handle-signals=no` with multiple threads (JULIA_NUM_THREADS > 1).\n"
         "This will cause segmentation faults due to GC safepoint failures.\n"
         "Remove `--handle-signals=no` or set JULIA_NUM_THREADS=1.\n"

